### DISCUSSION TITLE
Tesseract Python 0.1.7.post2 released (with Ubuntu 22.04 support)

### DISCUSSION THREAD

**Discussion by johnwason** [tesseract-robotics/tesseract#820] [General]
The Tesseract python packages have been updated to 0.1.7.post2. This is still an older version of the upstream tesseract sources, but fixes a few issues with the Python packages.

The main improvement is the addition of Linux packages for Python 3.7, 3.9, and 3.10. The Python 3.10 is compatible with Ubuntu 22.04.

To install on Windows:

```
python -m pip install tesseract-robotics tesseract-robotics-viewer
```

To install on Ubuntu 20.04 and Ubuntu 22.04:

```
sudo apt install python3-pip python3-numpy
# The supplied version of pip is too old for manylinux_2_31, on Ubuntu 20.04 upgrade pip
python3 -m pip install -U pip
python3 -m pip install --user  tesseract_robotics tesseract_robotics_viewer
```

**Comment by johnwason** [comment]
@Levi-Armstrong can we create a release for tesseract and tesseract_planning?

**Reply by Levi-Armstrong** [reply]
For the latest dev?

**Reply by johnwason** [reply]
I am going to do an update to the Python wrappers and would prefer to use a release version.
